doc/classes/EditorSettings.xml

```diff
@@ -207,6 +207,9 @@
 			If [code]true[/code], displays folders in the FileSystem dock's bottom pane when split mode is enabled. If [code]false[/code], only files will be displayed in the bottom pane. Split mode can be toggled by pressing the icon next to the [code]res://[/code] folder path.
 			[b]Note:[/b] This setting has no effect when split mode is disabled (which is the default).
 		</member>
+		<member name="docks/filesystem/other_file_extensions" type="String" setter="" getter="">
+			List of unsuppored file extensions to show in the FileSystem dock.
```

Comment:
typo: `unsupported `

Reply:
seems like this is still not fixed despite marked as resolved? The docs still say unsuppored here.